src/PowerShellEditorServices.Hosting/BuildInfo.cs

```diff
@@ -6,6 +6,6 @@ public static class BuildInfo
     {
         public static readonly string BuildVersion = "<development-build>";
         public static readonly string BuildOrigin = "<development>";
-        public static readonly System.DateTime? BuildTime = System.DateTime.Parse("2019-12-06T21:43:41", CultureInfo.InvariantCulture.DateTimeFormat);
+        public static readonly System.DateTime? BuildTime = System.DateTime.Parse("2020-05-03T16:39:35", CultureInfo.InvariantCulture.DateTimeFormat);
```

Comment:
Can you revert this? The build is supposed to mark this with git to not change... But it's not always worked.